[DOC] Installing ruby for Apple silicon missing (#8618)
opensearch-project/documentation-website: CONTRIBUTING.md

1. Install [Ruby](https://www.ruby-lang.org/en/) if you don't already have it. We recommend [RVM](https://rvm.io/), but you can use any method you prefer:

>    ###### For non-Apple silicon machines:

Reviewer: ```suggestion
    **For non-Apple silicon machines***
```

   ```
   curl -sSL https://get.rvm.io | bash -s stable
   rvm install 3.2.4
   ruby -v
   ```

>    ###### For Apple silicon machines: 

Reviewer: ```suggestion
   **For Apple silicon machines:** 
```

   Source: [Installing Ruby on Apple Silicon by Emmanuel Bernard ](https://emmanuelbernard.com/blog/2023/12/01/installing-ruby-on-apple-silicon/) 

   ```
   # Assumes Brew is installed
   curl -sSL https://get.rvm.io | bash -s stable

>    rvm install 3.2.4 --with-openssl-dir=$(brew --prefix openssl@1.1)

Reviewer: Can you indicate other versions of SSL besides 1.1? 3.3 is the current version, https://github.com/ruby/openssl/blob/master/README.md

Author: Okay what about:

   ```
   Please check with the current [SSL versions compatible with ruby](https://github.com/ruby/openssl/blob/master/README.md) 3.2.4 if you don't have `openssl@3`

   # Assumes Brew is installed
   curl -sSL https://get.rvm.io | bash -s stable
   rvm install 3.2.4 --with-openssl-dir=$(brew --prefix openssl@3)
```

Reviewer: We could always make the instruction version agnostic, for example:

```
# Assumes Brew is installed
curl -sSL https://get.rvm.io | bash -s stable
rvm install 3.2.4 --with-openssl-dir=$(brew --prefix openssl@<openssl-version>)
```

Author: Hey @Naarcha-AWS thank you for your persistence on this issue, I pushed a commit that addresses this concern

[f913275](https://github.com/opensearch-project/documentation-website/pull/8618/commits/f913275ed54145711ebf6582ee191ddaf6bc6f67)

Optionally, you can install the [Vale VSCode](https://github.com/chrischinchilla/vale-vscode) extension, which integrates Vale with Visual Studio Code. By default, only _errors_ and _warnings_ are underlined. To change the minimum alert level to include _suggestions_, go to **Vale VSCode** > **Extension Settings** and select **suggestion** in the **Vale > Vale CLI: Min Alert Level** dropdown list. 

## Troubleshooting

> This section details potential solutions for known issues.

Reviewer 2: ```suggestion
This section provides information about potential solutions for known issues.
```

> ### Installing Ruby on Apple Silicon machines

Reviewer 2: ```suggestion
### Installing Ruby on an Apple Silicon machine
```

> If you're having trouble installing Ruby with `rvm` on Apple Silicon machines, it could be to do an OpenSSL version misalignment. To fix this issue, use the following command, replacing `<openssl-version>` with your [desired version](https://github.com/ruby/openssl/blob/master/README.md):

Reviewer 2: ```suggestion
If you're having trouble installing Ruby with `rvm` on an Apple Silicon machine, it could be because of an OpenSSL version misalignment. To fix this issue, use the following command, replacing `<openssl-version>` with your [desired version](https://github.com/ruby/openssl/blob/master/README.md):
```
